lighthouse-core/lib/asset-saver.js

```diff
@@ -131,19 +131,20 @@ function prepareAssets(artifacts, results) {
 /**
  * Writes trace(s) and associated screenshot(s) to disk.
  * @param {!Artifacts} artifacts
- * @param {!Results} results
+ * @param {{path: string, title: string}} options
  * @return {!Promise}
  */
-function saveAssets(artifacts, results) {
-  return prepareAssets(artifacts, results).then(assets => {
+function saveAssets(artifacts, options = {}) {
```

Comment:
this doesn't appear to be treated as an optional parameter, so any need for the default?